### PR TITLE
Add new Transmission risk encoding configuration in preparation of ENF v2 adoption

### PR DESCRIPTION
**Pull request by EugenM-SAP**
## Checklist

* [x] Test your changes as thoroughly as possible before you commit them. Preferably, automate your test by unit/integration tests.
* [x] Make sure `mvn install` runs for the whole project and, if you touched any code in the respective service, submission and distribution service can be run with `spring-boot:run`

## Description

- This PR adds a configuration class that contains mappings which will be used to encode TRL in  DSOS and Report Type prior to distributing keys to the CDN. The intent is to allign with ENF v2 transmission risk calculations done on the client side. 
- The PR does not contain the actual invocation of these derivations (in a follow-up), only the mapping configurations.

### CODE REVIEW

File: services/distribution/src/main/resources/master-config/transmission-risk-encoding.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    8: 2
+  trl-to-rt:
+    1: 1
+    2: 2
+    3: 3
+    4: 4
+    5: 1
+    6: 2
+    7: 3
+    8: 4
```
Comment (hilmarf) [inline services/distribution/src/main/resources/master-config/transmission-risk-encoding.yaml:29]: @mlenkeit what do you think? All good?

**Review by hilmarf — APPROVED** [review]

File: services/distribution/src/main/resources/master-config/transmission-risk-encoding.yaml (hunk 1 above)
Comment (mlenkeit) [reply services/distribution/src/main/resources/master-config/transmission-risk-encoding.yaml:29]: LGTM 👍

Comment (mlenkeit) [reply services/distribution/src/main/resources/master-config/transmission-risk-encoding.yaml:29]: Are there any automated tests to check if these parameters are actually pulled?

Comment (EugenM-SAP) [reply services/distribution/src/main/resources/master-config/transmission-risk-encoding.yaml:29]: There are tests to validate the values, and there will be a follow-up PR with the code that actually performs the encoding by using this map. There we will have tests which cover the scenario you mention

**Review by sorin-stefan-iovita-sap — APPROVED** [review]

**Pull request merged**